NEWS.md

```diff
@@ -1,15 +1,176 @@
 next:
 
-- Update Gradle to 8.9 thanks to Christophe!
-- replace length = 0 checks with isEmpty() — thanks to Juiceman!
-- do not recommend disabling js helpers in fproxy (these are reviewed)
-- add more resilient plugin list exception handling
-- Remove gc meddling that hasn’t been necessary for many Java releases
-- Add webp filter, thanks to Torusrxxx!
-- Update Japanese localization, thanks to qupo1!
-- Fix build info with gradle daemon, thanks to Bombe!
+- 
+
+1499:
+
+# User changelog
+
+This release mainly brings improvements in five areas:
+
+- improved privacy protection
+- optimized and fixed networking layer
+- support for website authors
+- better user experience
+- clean ups and code purges
+
+## Privacy
+
+The privacy improvements are reduced pings and DNS lookups. These make
+it less likely that a node gets detected by pings it sends for other
+nodes that use DNS addresses to have permanent addresses. Also thanks
+to Torusrxxx SSL support is updated and when setting up a node for
+access via SSL, unencrypted HTTP links now get upgraded to secured
+HTTPS automatically.
+
+## Networking
+
+The networking layer got a fix for very fast nodes. On these the bulk
+queue could be starved when realtime requests were received faster
+than they could be completed. Also bertm fixed bulk backoff getting
+ignored.
+
+Juiceman removed unnecessary boxing and unboxing to reduce CPU load
+and Torusrxxx improved IPv6 handling.
+
+## Website
+
+Torusrxxx also added webp support to the content sanitation (filters),
```

Comment:
Need to mention animation and lossless WebP are not supported yet.

Reply:
fixed (along with the other points) for the article for the website. Thank you!

Reply:
⇒ https://github.com/hyphanet/website/commit/d83203a764dc41a3873a1c88c31301c56bcdac4b

Reply:
It can help if you add a news entry for changes, but the list format is enough. I format it in a multi step process for the website: first in NEWS as rough article, then I turn that into a real article in the website repo. For writing that article it’s necessary to know all changes that will go into the release.

Easier than putting it into the NEWS for each PR (which would create merge conflicts) is to ensure that the commit messages can easily be used as base for a news entry. For example putting the different tags into the commit message (even if that makes the commit message longer).